puppet/puppetmaster-bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+PUPDIR=/etc/puppetlabs/code
+ENVDIR=${PUPDIR}/environments/production
+
+if dpkg-query -l puppetserver > /dev/null; then
```

Comment:
It's a dependency of `puppetserver`, so if it's not installed it'll be pulled in with that anyway. The idea is that we have a common `puppet-bootstrap.sh` that runs on all machines and then an extra `puppetmaster-bootstrap.sh` that runs on the `boot` machine afterwards.